ISISComputingGroup/ibex_gui
# Ticket 5425: Choose Script Generator Save Location

### Description of work

Adds a file dialog to select where the generated script should be saved.

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/5425

To test:
* Confirm that you can save a generated script
* Confirm that you are prompted if overwriting a script
* Confirm that you are prompted to open in an editor and that you can do so

### Unit tests

* Confirm modified tests still work

### System tests

See https://github.com/ISISComputingGroup/System_Tests_UI_E4/pull/52

---

#### Code Review


### Final Steps
